manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "id" : "com.github.bazylevnik0.Convolution",
```

Comment:
as per the documentation it should be `io.github`

Reply:
also technically since it is not hosted on github, there might be questions.

Reply:
`org.gnome.gitlab` is the base

one doesn't simply make up IDs. `com.github` will be rejected anyway.

Reply:
gitlab is NOT a username. So the latter.